Documentation/HowToGuides/ConvertingAFloatActionToABooleanAction/README.md

```diff
@@ -0,0 +1,55 @@
+# Converting A Float Action To A Boolean Action
+
+> * Level: Beginner
+>
+> * Reading Time: 5 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+Sometimes you may want to use the values from a FloatAction to activate or deactivate an output type. For example, pressing down the trigger button axis on a controller causes an output to occur when the trigger is fully pressed down.
+
+The `Input Unity Input Manager Axis Actions' provide a way of getting the varying value data from an axis but don’t really provide a way of letting us trigger something when that value reaches a specific value. This can be done by taking the float value and converting it into a boolean when the float value is within our specified bounds. We can then take this boolean value and pass it into a Boolean Action for hooking up events that are either on or off (true/false).
+
+We can do this using the Data Type Converters provided with Zinnia.Unity which allow conversions from one DataType to another.
+
+## Let's Start
+
+### Step 1
+
+Create a new `Cube` Unity 3D Object by selecting `Main Menu -> GameObject -> 3D Object -> Cube` and change the `Transform` properties to:
+
+* Position: `X = 0`, `Y = 0`, `Z = 0`
+
+![Adding A Cube](assets/images/AddingACube.png)
+
+### Step 2
+
+Drag and drop the `Input Unity Input Manager 1D Axis Action` and change its Axis Name to represent that of the Xbox Controller's left trigger, which is `Tilia.Input.UnityInputManager_Axis9`.
+
+![Adding Input Unity Input Manager 1D Axis Action] (assets/images/AddingInputUnityInputManager1DAxisAction.png)
+
+### Step 3 
+
+Create an `Empty GameObject` Unity 3D Object by selecting `Main Menu -> GameObject -> Create Empty`.
+
+Select `Add Component` then add a `boolean action`. Add an 'Activated' and `Deactivated`. Drag and drop the `Cube` into the `Activated` and change the function from `no function` -> `GameObject` -> `Static Parameters` -> `SetActive(bool)`. Repeat this for the `Deactivated` and tick the box next to where we have drag and dropped the `Cube`.
+
+![Adding A Boolean] (assets/images/AddingABoolean.png)
+
+### Step 4
+
+Add a `Float to Boolean` action, press the `+` button to add a transform and drag and drop the empty `GameObject` into the transform. Select `No Function -> BooleanAction -> Receive` then set the `Positive Bounds` values to `0.5` and `1`.
+
+![Adding Float To Boolean] (assets/images/AddingFloatToBoolean.png)
+
+### Step 5
+
+Add a `Value Change` and drag and drop the GameObject inside, the set `No Function -> FloatToBoolean -> Do Transform`
+
+![Value Change] (assets/images/ValueChange.png)
+
+### Done
+
+Now you have a `Boolean Action` that will emit `true` when the Trigger axis is squeezed at least 50% and will emit `false` when is no longer squeezed as much. This `Boolean Action` can now be used to drive other output events too such as grabbing an item or even activating a pointer.
```

Comment:
Mention something here about playing the scene and pressing the trigger, something like:


Play the Unity scene and press the Left Trigger axis on the Xbox Controller and notice how the Cube will disappear when the axis passes half way. This `Boolean Action` can now be used to drive other output events too such as grabbing an item or even activating a pointer.